### PR TITLE
Fixes a bug with in processing annonymous schemas

### PR DESCRIPTION
**Pull request by piotrswigon**

### CODE REVIEW

File: src/runtime/tests/schema-test.ts [hunk 1]
```diff
@@ -443,4 +443,25 @@ describe('schema', () => {
     assert.strictEqual(schema.fields.age.type, 'Number');
     assert.strictEqual(schema.fields.custom.type, 'Bytes');
   });
+
+  it('parses annonymous schemas', async () => {
```
Comment (Cypher1) [inline src/runtime/tests/schema-test.ts]: This schema is named (X) but has no fields, is it anonymous or empty?

File: src/runtime/tests/schema-test.ts [hunk 2]
```diff
@@ -443,4 +443,25 @@ describe('schema', () => {
...
+    const manifest = await Manifest.parse(`alias schema * as X`);
+    const schema = manifest.findSchemaByName('X');
+
+    assert.isEmpty(schema.names);
+    assert.isEmpty(schema.fields);
+  });
+
+  it('parses annonymous inline schemas', async () => {
```
Comment (Cypher1) [inline src/runtime/tests/schema-test.ts]: nit: typo: anonymous.

File: src/runtime/tests/schema-test.ts (hunk 1 above)
Comment (piotrswigon) [reply src/runtime/tests/schema-test.ts]: Schema has no names. 'X' is an alias for referencing this schema.

File: src/runtime/tests/schema-test.ts (hunk 2 above)
Comment (piotrswigon) [reply src/runtime/tests/schema-test.ts]: Done, thanks!

File: src/runtime/tests/schema-test.ts (hunk 1 above)
Comment (Cypher1) [reply src/runtime/tests/schema-test.ts]: Sounds good

**Pull request merged**